### PR TITLE
#368 - deploy to production

### LINKED ISSUE #368: Deploy to production
So, we are ready to go to production with new page design :rocket: 

The new page should be on address like previous page. Also we should make copy of beta database and move data to production database

### PR DESCRIPTION
**Pull request by Blusia**
This PR contains files for implementing the production version of the Blumilk company website on the server.

This should close #368

### CODE REVIEW

File: environment/prod/deployment/prod/.env.prod [hunk 1]
```diff
@@ -0,0 +1,29 @@
+COMPOSE_PROJECT_NAME=blumilk-website-prod
+
+BLUMILK_WEBSITE_HOST_NAME=blumilk.test.pl
```
Comment (mtracz) [inline environment/prod/deployment/prod/.env.prod:3]: Should be changed. 
We can use temporary domain for test purposes like `prod.website.blumilk.pl` (this need to be added to the DNS provider), then migrate data and check. If everything will be fine, we will only switch domains.

File: environment/prod/deployment/prod/.env.prod [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+REDIS_HOST=blumilk-website-prod-redis
+REDIS_PORT=6379
+
+MAIL_MAILER=smtp
+MAIL_HOST=smtp.gmail.com
+MAIL_PORT=465
+MAIL_ENCRYPTION=tls
+MAIL_FROM_ADDRESS=no-reply@blumilk.pl
+MAIL_FROM_NAME="${APP_NAME}"
```
Comment (mtracz) [inline environment/prod/deployment/prod/.env.prod:29]: I think it should be adjusted also, because `blumilk-website` doesnt look good.

Comment (Blusia) [reply environment/prod/deployment/prod/.env.prod]: Changed.

File: resources/views/email.blade.php [hunk 3]
```diff
@@ -196,7 +197,7 @@
                 <footer>
                     <div class="footer-top">
                         <div class="footer-logo">
-                            <img src="{{ asset('identification/logo.svg') }}" alt="Blumilk logo"/>
+                            <img src="https://blumilk.pl/identification/logo.svg" alt="Blumilk logo"/>
```
Comment (mtracz) [inline resources/views/email.blade.php:200]: Should be revert to `asset` which automatically uses APP_URL.
Image has been blocked by Authelia.
I propose to set Authelia only for admin panel for beta too.

File: resources/views/email.blade.php [hunk 4]
```diff
@@ -202,23 +206,25 @@
                 <footer>
                     <div class="footer-top">
                         <div class="footer-logo">
-                            <img src="{{ asset('identification/logo.png') }}" alt="Blumilk logo"/>
+                            <a href="https://blumilk.pl" class="social-link" target="_blank">
```
Comment (mtracz) [inline resources/views/email.blade.php:209]: I would like to replace 
`href="https://blumilk.pl"`
with
`href="{{ config(app.url) }}"`

Comment (EwelinaSkrzypacz) [reply resources/views/email.blade.php]: Fixed

**Review by EwelinaSkrzypacz — DISMISSED** [review]

**Review by mtracz — APPROVED** [review]

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Pull request merged**